Test/exceptions/Exceptions1.dfy

```diff
@@ -0,0 +1,79 @@
+// RUN: %dafny /compile:3 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+include "./NatOutcome.dfy"
```

Comment:
This is okay, but it's not what first would have come to mind for me. I would have merged Exception1.dfy and NetOutcome.dfy into one file, which would keep the test files more independent of each other.

Reply:
I did this because I want to reuse `NatOutcome.dfy` in several files which fail after different stages.